app/controllers/api_controller/tags.rb

```diff
@@ -44,10 +44,7 @@ def tags_delete_resource(_parent, _type, id, data)
     end
 
     def create_resource_tags(_type, _id, data)
-      if data.key?("id") || data.key?("href")
-        raise BadRequestError,
-              "Resource id or href should not be specified for creating a new tag resource"
-      end
+      assert_id_not_specified(data, 'tag resource')
```

Comment:
this can just be (data, type) to be consistent with others, and change signature _type to type.  Thanks.